smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -101,8 +101,13 @@ class SwiftWriter(private val fullPackageName: String) : CodeWriter() {
         }
     }
 
-    fun addImport(packageName: String, isTestable: Boolean = false) {
-        imports.addImport(packageName, isTestable)
+    fun addImport(target: String, isTestable: Boolean = false) {
+        imports.addImport(target, isTestable)
+    }
+
+    fun addImport(dep: SymbolDependencyContainer, target: String) {
+        dependencies.addAll(dep.dependencies)
+        imports.addImport(target)
     }
```

Comment:
This function reads weird.  Seems like if we need access to `dependencies`, we should create a function called:
```
fun addDependencies(dependencies: List<SymbolDependency>) {
}
```